addons/settings/BW_Settings.inc.sqf

```diff
@@ -42,6 +42,8 @@ _settings = [
 
 [QACEGVAR(advanced_fatigue,performanceFactor), 1.4],
 [QACEGVAR(advanced_fatigue,recoveryFactor), 1.6],
+[QACEGVAR(advanced_fatigue,terraingradientfactor), 0.4],
```

Comment:
this is new in ace 3.18, it makes walking on slopes and uneven terrain much more exhausting 
IIRC on first mission everyone was getting tired too quickly so we just changed it on the fly to this
I'm just copying what was saved

Reply:
was me, during that TVT we were walking directly uphill so i bumped it down pretty big, it'll be worth a re-look later, but these vals are fine for now